pom.xml

```diff
@@ -13,8 +14,7 @@
 	<description>Aggregate Test and Job Results in a single HTML/email report.</description>
 	<url>https://github.com/jenkinsci/test-results-aggregator-plugin</url>
 	<properties>
-		<jenkins.version>2.277</jenkins.version>
-		<java.level>8</java.level>

```

Comment:
A recent plugin parent POM update has made this an unused variable; following its release notes, deleting the dead code.